runelite-client/src/test/java/net/runelite/client/plugins/wintertodt/WintertodtPluginTest.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2019, Adam <Adam@sigterm.info>
```

Comment:
```suggestion
 * Copyright (c) 2019, Kusha Gharahi<kusha.me>
```

@Adam- I would like to claim the copyright here, if possible please.

Reply:
So add yourself to it.